tests/Transports.AspNetCore.Tests/AuthorizationHelperTests.cs

```diff
@@ -68,7 +68,7 @@ public async Task Authorize(bool expected)
         ret.ShouldBe(expected);
 
         bool ranHandler = false;
-        ret = await AuthorizationHelper.AuthorizeAsync(new AuthorizationParameters<object?>(mockContext.Object, options, async _ => { ranHandler = true; }, null, null), null);
+        ret = await AuthorizationHelper.AuthorizeAsync(new AuthorizationParameters<object?>(mockContext.Object, options, _ => { ranHandler = true; return Task.CompletedTask; }, null, null), null);
```

Comment:
I just fixed compiler warning. What is bad?

Reply:
it is harder to read.  i don't want all the tests to have `return Task.CompletedTask` in there.  we can do that in production code to optimize performance.  but it just makes the code harder to read unnecessarily.

Reply:
>  i don't want all the tests to have return `Task.CompletedTask` in there

Not all. Only those that have `async` without `await`. Only 3 places for 3465 tests. Are you serious? Really?

Reply:
Yes really.  You're the one that is trying to add analyzers here, not me.  We do not need analyzers here.  The code is just fine as-is.  What we need is review on code-first federation support.  I've written a LOT of tests where this is a real pain, and every time I turn around I step on this stupid warning, where it might be 3 times PER TEST, not just 3 total.  Perhaps here there's only 3 places in this project.  Bad code design is bad code design no matter if it's 3 or 300.

Reply:
>  We do not need analyzers here. The code is just fine as-is.

This is only a subjective judgment. It is not based on formal rules, but only on emotions and mood. If you are so important to go against the recommendations of the MS here... OK. Reverted.